client/src/components/Navbar/Navbar.tsx

```diff
@@ -32,63 +41,82 @@ const Navbar: React.FC = () => {
     handleClose();
     logout();
   };
-
+  console.log(loggedInUser);
   return (
-    <AppBar position="static">
-      <Toolbar>
-        <IconButton size="large" edge="start" color="inherit" aria-label="menu" sx={{ mr: 2 }}>
-          <MenuIcon />
-        </IconButton>
-        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
-          My App
-        </Typography>
+    <AppBar className={classes.root} position="static">
+      <Toolbar className={classes.toolbar}>
+        <Link to={'/home'}>
+          <img src={logo} alt="Pet Sitter Logo"></img>
+        </Link>
+        {!loggedInUser && (
+          <>
+            <Box className={classes.flexContainer}>
+              <Link className={classes.link} to={'/'}>
+                BECOME A SITTER
+              </Link>
+              <Link className={classes.buttonLink} to={'/login'}>
+                <Button type="submit" className={classes.login}>
+                  LOGIN
+                </Button>
+              </Link>
+              <Link className={classes.buttonLink} to={'/signup'}>
+                <Button type="submit" size="large" variant="contained" className={classes.signup}>
```

Comment:
```suggestion
                <Button size="large" variant="contained" className={classes.signup}>
```
Having this Button as `type="submit"` does not seem appropriate here